C++/C++.sublime-syntax

```diff
@@ -87,6 +87,16 @@ contexts:
   unique-keywords:
     - match: \busing\b
       scope: keyword.control.c++
+    - match: \bbreak\b
+      scope: keyword.control.flow.break.c++
+    - match: \bcontinue\b
+      scope: keyword.control.flow.continue.c++
+    - match: \bgoto\b
+      scope: keyword.control.flow.goto.c++
+    - match: \breturn\b
+      scope: keyword.control.flow.return.c++
+    - match: \bthrow\b
+      scope: keyword.control.flow.throw.c++
     - match: \b({{control_keywords}})\b
```

Comment:
Could the `control_keywords` variable be made simpler now? It might be used elsewhere for different purposes, so I don't know.

Reply:
How do you mean? The `control_keywords` variable is indeed used elsewhere so these flow keywords are still in it, it's just in this one context that they get some special pre-`control_keywords` matching.